locales/en.yml

```diff
@@ -1,14 +0,0 @@
-button:
-  about: About
-  back: Back
-  go: GO
-  home: Home
-  toggle_dark: Toggle dark mode
-  toggle_langs: Change languages
-intro:
-  desc: Opinionated Vite Starter Template
-  dynamic-route: Demo of dynamic route
-  hi: Hi, {name}!
-  aka: Also known as
-  whats-your-name: What's your name?
-not-found: Not found
```

Comment:
I think you replaced this file to `pl.yml` but we still need the original `en.yml` file, can you revert this change?

Reply:
Sorry i did it on my phone, so that's why en.yml was accidently removed